### ISSUE TITLE
Explore adding support for snapshot releases to the openvpn package

### ISSUE THREAD

**Issue by dgalbraith** [dgalbraith/chocolatey-packages#579]
The  [OpenVPN/openvpn-build](https://github.com/OpenVPN/openvpn-build ) project produces snapshot builds of the OpenVPN Windows installers.  Explore updating the openvpn package to incorporate support for the snapshot releases.

**Comment by dgalbraith** [comment]
@mattock perhaps we can flesh out requirements and an approach here.

**Comment by mattock** [comment]
Hi @dgalbraith! Yeah, this will work. I checked the GitHub Actions [Workflow file](https://github.com/OpenVPN/openvpn-build/actions/runs/11103822955/workflow#L65) and it does not do signing (-sign parameter is missing). So I'll start getting that implemented - otherwise the MSI files would be practically useless.

Would, in your opinion, a simple web server directory with MSIs be enough at the Chocolatey end? With the latest MSIs showing up at the top, or possibly stored in a separate directory.

**Comment by dgalbraith** [comment]
@mattock yes.  A simple web server directory with MSIs would be quite simple to integrate into the Chocolatey  package to provide interim snapshot release builds.
